### DISCUSSION TITLE
Is it possible to save a full model with the adapters activated?

### DISCUSSION THREAD

**Discussion by vabatta** [adapter-hub/adapters#632] [Q&A]
I would like to save my pre-trained adapters with the model to later load them into an ONNX runtime environment (as I don't think it would be possible with transformers.js to load them adapters at the moment).

Is this possible somehow?

**Comment by vabatta** [comment]
Is there any way to save it as a full PyTorch model so that I can convert it to ONNX?

**Comment by calpt** [comment]
Unortunately, onnx export is not supported at this point. This is tracked in #224.
